data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/model/EnvironmentRevision.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservicemodel.v1.model;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public class EnvironmentRevision {
```

Comment:
Is this a public type? Where is this used?

Reply:
See the next commit with the core schedulers, it's used by the inner scheduler loop.